restonomer-docs/source/restonomer_response_handler.md

```diff
@@ -0,0 +1,54 @@
+Restonomer can handle various api response data format.
+Below are some types along with a sample config.
+
+# Json
+
+The user needs to specify the data type as JSON in the body-format section in data-response.
+
+```hocon
+name = "checkpoint_json_handler"
+
+data = {
+  data-request = {
+    url = "http://localhost:8080/add-column-transformation"
+  }
+
+  data-response = {
+    body-format = "JSON"
+
+    transformations = [
+      {
+        type = "add-literal-column"
+        column-name = "col_D"
+        column-value = "val_D"
+        column-data-type = "string"
+      }]
+  }
```

Comment:
I see that in this example you are missing persistence but persistence is mandatory. This example is not accurate. I suggest to remove transformations and add persistence as transformations is optional.

Reply:
resolved